webdriver/tests/sessions/get_timeouts.py

```diff
@@ -0,0 +1,38 @@
+from tests.support.asserts import assert_success
+
+
+def get_timeouts(session):
+    return session.transport.send("GET", "session/{session_id}/timeouts"
+                                  .format(session_id=session.session_id))
+
+
+# 8.4 Get Timeouts
+
+def test_get_timeouts(session):
```

Comment:
Can you split this into three tests? One for checking the payload (up to line 26), one for checking the session’s default values (lines 28 thru 30), and one for checking that _Get Timeouts_ returns the right values after having invoked `session.timeouts.script = 42`.

Reply:
fixed